pkgs/build-support/docker/default.nix

```diff
@@ -434,6 +434,8 @@ rec {
         if [[ -n "$fromImage" ]]; then
           echo "Unpacking base image..."
           tar -C image -xpf "$fromImage"
+          # Do not import the base image configuration
+          rm -f image/*.json
```

Comment:
For instance, this is the structure of the image of ubuntu + bash:
```
tar -tf /nix/store/91vm2lng91f313bnngi87qg3grxynjk0-docker-image-bashOnUbuntu.tar.gz
./
./ef590a0764fab1ffa7a6b08499de5033937c003a283af521371b69a8e72a11bd/
./ef590a0764fab1ffa7a6b08499de5033937c003a283af521371b69a8e72a11bd/layer.tar
./ef590a0764fab1ffa7a6b08499de5033937c003a283af521371b69a8e72a11bd/VERSION
./ef590a0764fab1ffa7a6b08499de5033937c003a283af521371b69a8e72a11bd/json
./1fdb68b6ffa4ad4ae4bfacdbfd6595ef9128d0379eb1a6b36470ac1ec547e40a/
./1fdb68b6ffa4ad4ae4bfacdbfd6595ef9128d0379eb1a6b36470ac1ec547e40a/layer.tar
./1fdb68b6ffa4ad4ae4bfacdbfd6595ef9128d0379eb1a6b36470ac1ec547e40a/VERSION
./1fdb68b6ffa4ad4ae4bfacdbfd6595ef9128d0379eb1a6b36470ac1ec547e40a/json
./54333f1de4ed2730bea18e49605b2ea8f8a2689db213ece94db6ccbc8cf279a6.json
./repositories
./7dd07830e2ffed07a198f014fa9c272d2ca898a9dce00919e0662db7635e80fd/
./7dd07830e2ffed07a198f014fa9c272d2ca898a9dce00919e0662db7635e80fd/layer.tar
./7dd07830e2ffed07a198f014fa9c272d2ca898a9dce00919e0662db7635e80fd/VERSION
./7dd07830e2ffed07a198f014fa9c272d2ca898a9dce00919e0662db7635e80fd/json
./b72351754cb42e486d4b35699b73170a9e7560001abe727de5ea376adcb66c8b/
./b72351754cb42e486d4b35699b73170a9e7560001abe727de5ea376adcb66c8b/layer.tar
./b72351754cb42e486d4b35699b73170a9e7560001abe727de5ea376adcb66c8b/VERSION
./b72351754cb42e486d4b35699b73170a9e7560001abe727de5ea376adcb66c8b/json
./5c0fae2698664beefe0c37cf9815bc8173afc87965bec5455dd4addbb2d23847/
./5c0fae2698664beefe0c37cf9815bc8173afc87965bec5455dd4addbb2d23847/layer.tar
./5c0fae2698664beefe0c37cf9815bc8173afc87965bec5455dd4addbb2d23847/VERSION
./5c0fae2698664beefe0c37cf9815bc8173afc87965bec5455dd4addbb2d23847/json
./manifest.json
./c4dd515170301a4220cf3006bd541bb22dd4bddebe0297f8cd42e28c2667d8ef/
./c4dd515170301a4220cf3006bd541bb22dd4bddebe0297f8cd42e28c2667d8ef/layer.tar
./c4dd515170301a4220cf3006bd541bb22dd4bddebe0297f8cd42e28c2667d8ef/VERSION
./c4dd515170301a4220cf3006bd541bb22dd4bddebe0297f8cd42e28c2667d8ef/json
```

What would be deleted by this patch is the file `54333f1de4ed2730bea18e49605b2ea8f8a2689db213ece94db6ccbc8cf279a6.json`.
But the file `manifest.json` is also wrong since it is inherited from the base image.
So, I think both of these files could be deleted.